spec/prog/vnet/load_balancer_nexus_spec.rb

```diff
@@ -21,6 +21,23 @@
     dz
   }
 
+  def create_vm_with_ips(ipv4: false, ipv6: false)
+    @vm_counter ||= 0
+    @vm_counter += 1
+    vm = Prog::Vm::Nexus.assemble("pub key", ps.project_id, name: "dns-vm-#{@vm_counter}", private_subnet_id: ps.id).subject
+    if ipv6
+      # Set ephemeral_net6 to produce ip6_string via .nth(2)
+      prefix = format("fd10:9b0b:6b4b:%04x::/79", @vm_counter)
+      vm.update(ephemeral_net6: prefix)
+    end
+    if ipv4
+      # Generate unique IPv4 address
+      ip = format("10.0.%d.%d", @vm_counter / 256, @vm_counter % 256)
```

Comment:
This seems over engineered, unless a single spec is actually creating more than 256 VMs. That said, better to a single calculation instead of two.

```suggestion
      ip = format("10.0.%d.%d", *@vm_counter.divmod(256))
```